com.microsoft.mrtk.graphicstools.unity/Samples~/Experimental/Magnifier/MagnifierShowcase.unity

```diff
@@ -0,0 +1,1954 @@
+%YAML 1.1
```

Comment:
Looks like something may have broken in this scene? I don't see magnification anymore.